{{ cookiecutter.package_name|slugify }}/Dockerfile

```diff
@@ -35,32 +56,13 @@ RUN --mount=type=cache,id=apt-cache,target=/var/cache/apt/ \
     echo 'eval "$(starship init zsh)"' >> ~/.zshrc && \
     zsh -c 'source ~/.zshrc'
 
-# Install the development Python environment.
-COPY .pre-commit-config.yaml poetry.lock* pyproject.toml /app/
-RUN --mount=type=cache,id=poetry,target=/root/.cache/ \
-    {%- if cookiecutter.private_package_repository_name %}
-    --mount=type=secret,id=poetry_auth,target=/root/.config/pypoetry/auth.toml \
-    {%- endif %}
-    mkdir -p src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/ && touch src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/__init__.py && touch README.md && \
-    poetry install --no-interaction && \
-    mkdir -p /var/lib/poetry/ && cp poetry.lock /var/lib/poetry/ && \
+# Persist output generated during docker build so that we can restore it in the dev container.
+RUN mkdir -p /var/lib/poetry/ && cp poetry.lock /var/lib/poetry/ && \
     git init && pre-commit install --install-hooks && \
     mkdir -p /var/lib/git/ && cp .git/hooks/commit-msg .git/hooks/pre-commit /var/lib/git/
 
-FROM base as ci
-
-# Install the run time Python environment.
-# TODO: Replace `--no-dev` with `--without test` when Poetry 1.2.0 is released.
-COPY poetry.lock pyproject.toml /app/
-RUN --mount=type=cache,id=poetry,target=/root/.cache/ \
-    {%- if cookiecutter.private_package_repository_name %}
-    --mount=type=secret,id=poetry_auth,target=/root/.config/pypoetry/auth.toml \
-    {%- endif %}
-    mkdir -p src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/ && touch src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/__init__.py && touch README.md && \
-    poetry install --no-dev --no-interaction
 {%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int or cookiecutter.with_typer_cli|int %}
```

Comment:
Nit-pick, but the blank line above this line should be removed and a blank line below this line should be added so that the two stages don't 'stick' to each other when rendered by cookiecutter.

Reply:
Fixed